test/transcoding/IntelFPGAMemoryAccesses.ll

```diff
@@ -0,0 +1,218 @@
+; LLVM IR generated by Intel SYCL Clang compiler (https://github.com/intel/llvm)
+; SYCL source code can be found below:
+
+; struct State {
+;  float Field1;
+;  int Field2;
+;  };
+;
+; void foo(float *A, int *B, State *C) {
+;   float *x;
+;   int *y;
+;   State *z;
+;   x = __builtin_intel_fpga_mem(A, 1 | 2, 0);
```

Comment:
Please add comments what are 1, 2, 4 and 8 here.

Reply:
As another suggestion, you can declare an `enum` or use `#define` to avoid using magic numbers

Reply:
Gone with the `#define` variant, thanks for the suggestion!